app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
```

Comment:
Look at the [PEP](https://peps.python.org/pep-0328/#rationale-for-parentheses) for more beautiful import examples.